.github/actions/e2e-deploy-vald-readreplica/action.yaml

```diff
@@ -87,12 +85,15 @@ runs:
       run: |
         make k8s/vald-readreplica/deploy HELM_VALUES=${VALUES} HELM_EXTRA_OPTIONS="${HELM_EXTRA_OPTIONS}"
 
+        stern vald-discoverer &
+        stern vald-lb-gateway &
+        stern vald-agent-readreplica &
```

Comment:
Stern usually outputs sizable logs, and it would not be easy to debug them on the GitHub Actions UI when they are logged out in the background.

How about setting up an area in another branch where GitHub Actions logs are aggregated and committed, and record all the results of stern as a file?　
(example: branch=github/actions/log, dir=$(ROOTDIR)/{workflow name}/{date}/{time hour only}/{workflow id})

There is a similar branch, gh-pages, which is used to store documentation, but it does not follow the original branch at all, but you can push with the Actions permission.

In particular, you could make it so that it commits only when a job fails.